google/cloud/spanner_v1/database.py

```diff
@@ -1209,7 +1235,7 @@ def __exit__(self, exc_type, exc_val, exc_tb):
                     request_options=self._request_options,
                     max_commit_delay=self._max_commit_delay,
                     exclude_txn_from_change_streams=self._exclude_txn_from_change_streams,
-                    **self._kw,
```

Comment:
Should this really be removed?

Reply:
This was removed by mistake. Added it back